.cloud-build/notebook-execution-test-cloudbuild.yaml

```diff
@@ -15,26 +15,24 @@ steps:
     args:
     - -c
     - 'python3 .cloud-build/CheckPythonVersion.py'
-  # Upgrade pip
-  - name: ${_PYTHON_IMAGE}
-    entrypoint: pip
-    args: ['install', '--upgrade', 'pip']
   # Fetch base branch if required
   - name: ${_PYTHON_IMAGE}
     entrypoint: /bin/sh
     args:
     - -c
     - 'if [ -n "${_BASE_BRANCH}" ]; then git fetch origin "${_BASE_BRANCH}":refs/remotes/origin/"${_BASE_BRANCH}"; else echo "Skipping fetch."; fi'
-  # Install Python dependencies
+  # Install Python dependencies  
```

Comment:
extraneous whitespace

Reply:
Got it. Will push the change after this round of CI finishes.